jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_PREP_OBS

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+export STRICT="NO"
+source "${HOMEgfs}/ush/preamble.sh"
+source "${HOMEgfs}/ush/jjob_header.sh" -e "ocnanalprepobs" -c "base ocnanal ocnanalprepobs"
+
+
+##############################################
+# Set variables used in the script
+##############################################
+export CDUMP=${CDUMP:-${RUN:-"gfs"}}
+# Ignore possible spelling error (nothing is misspelled)
+# shellcheck disable=SC2153
+
+##############################################
+# Begin JOB SPECIFIC work
+##############################################
+
+# Add UFSDA to PYTHONPATH
+export PYTHONPATH=${HOMEgfs}/sorc/gdas.cd/ush/:${HOMEgfs}/sorc/gdas.cd/ush/eva:${PYTHONPATH}
+export PYTHONPATH=${HOMEgfs}/sorc/gdas.cd/ush/soca:${PYTHONPATH}
```

Comment:
This should be done in the rocoto job in `jobs/rocoto/prepoceanobs.sh` and not in a j-job.
```suggestion
```